ppr-ui/src/components/collateral/Collateral.vue

```diff
@@ -95,376 +54,167 @@
-                    <v-menu offset-y left nudge-bottom="4">
-                      <template v-slot:activator="{ on }">
-                        <v-btn
-                          text
-                          small
-                          v-on="on"
-                          color="primary"
-                          class="actions__more-actions__btn"
-                          :disabled="addEditInProgress"
-                        >
-                          <v-icon>mdi-menu-down</v-icon>
-                        </v-btn>
-                      </template>
-                      <v-list class="actions__more-actions">
-                        <v-list-item @click="removeVehicle(row.index)">
-                          <v-list-item-subtitle>
-                            <v-icon small>mdi-delete</v-icon>
-                            <span class="ml-1">Remove</span>
-                          </v-list-item-subtitle>
-                        </v-list-item>
-                      </v-list>
-                    </v-menu>
-                  </span>
-                </div>
-              </td>
-            </tr>
-
-            <!-- Edit Form -->
-            <tr v-if="showEditVehicle[row.index]">
-              <td :colspan="getNumCols">
-                <v-expand-transition>
-                  <div class="edit-vehicle-container col-12">
-                    <edit-collateral
-                      :activeIndex="activeIndex"
-                      :invalidSection="invalidSection"
-                      @removeVehicle="removeVehicle"
-                      @resetEvent="resetData"
-                    />
-                  </div>
-                </v-expand-transition>
-              </td>
-            </tr>
-          </template>
-        </v-data-table>
-      </v-col>
-    </v-row>
-    <v-row class="pt-8" v-if="hasGeneralCollateral()">
-      <v-col>
-        <v-card
-          flat
-          id="general-collateral"
-          :class="{ 'invalid-message': showErrorComponent }"
-        >
-          <v-container fluid no-gutters class="pa-0">
-            <v-row no-gutters class="py-6">
-              <v-col cols="3" class="generic-label pa-4">
-                General Collateral
-              </v-col>
-              <v-col cols="9" class="pr-4">
-                <v-textarea
-                  v-model="generalCollateralDesc"
-                  id="generalCollateral"
-                  auto-grow
-                  counter="4000"
-                  filled
-                  label="Description of General Collateral"
-                  class="white pt-2 text-input-field"
-                  @keyup="validateGeneral()"
-                  :error-messages="generalCollateralError"
-                >
-                </v-textarea>
-              </v-col>
-            </v-row>
-          </v-container>
-        </v-card>
-      </v-col>
-    </v-row>
+    <vehicle-collateral :isSummary="false" :showInvalid="showInvalid && !valid" />
```

Comment:
Do we only want to show this if hasVehicleCollateral() ?

Reply:
That's handled within the vehicle collateral component